### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Update `powersimdata` following release to `0.5.5` in **Pipfile** and **requirements.txt**. A new **Pipfile.lock** is also generated

### What the code is doing
N/A

### Testing
Existing unit tests.

### Where to look
I had to update a couple of plotting procedure that display curtailable resources.

### Usage Example/Visuals
There was an issue with `plot+generation_ts_stack` where **Demand** and **Net Demand** where overwriting **Nuclear** and **Coal** and the hatch boxes had `None` as legend. By assigning  **Demand** and **Net Demand** to the last 2 entries of `labels` and not shifting the entries I was able to solve the issue. Tee only difference with the plot generated with our [previous version](https://breakthrough-energy.github.io/docs/postreise/index.html#id3) is that  **Demand** and **Net Demand** sit at the top of the legend:
Western

### Time estimate
10min

### CODE REVIEW

**Comment by jenhagg** [comment]
We should also update the powersimdata version in setup.cfg. Could also bump the postreise version unless you wanna do that separately.

**Comment by rouille** [comment]
> We should also update the powersimdata version in setup.cfg. Could also bump the postreise version unless you wanna do that separately.

I will bump `postreise` in a followup PR, I want to make sure first that we don't have other issues with the plots (we probably want to run the notebook you created).

File: postreise/plot/plot_bar_generation_stack.py [hunk 1]
```diff
@@ -98,13 +98,17 @@ def plot_bar_generation_stack(
     mi = ModelImmutables(s_list[0].info["grid_model"])
     type2color = mi.plants["type2color"]
     type2label = mi.plants["type2label"]
-    type2hatchcolor = mi.plants["type2hatchcolor"]
-    if t2c:
-        type2color.update(t2c)
-    if t2l:
-        type2label.update(t2l)
-    if t2hc:
-        type2hatchcolor.update(t2hc)
```
Comment (BainanXia) [inline postreise/plot/plot_bar_generation_stack.py:107]: I guess we still want to keep these updates for user specified color/label, right?

Comment (rouille) [reply postreise/plot/plot_bar_generation_stack.py:107]: Yes. Not sure why I removed these lines.

Comment (rouille) [reply postreise/plot/plot_bar_generation_stack.py]: Fixed

**Review by BainanXia — APPROVED** [review]
Thanks for taking care of these!

**Comment by rouille** [comment]
I think it is good to go now. After updating `plot_curtailment_ts`, I was able to successfully run everything in the [tutorial](https://breakthrough-energy.github.io/docs/postreise/index.html).

**Pull request merged**